src/Common/HashTable/HashTable.h

```diff
@@ -1263,30 +1251,6 @@ class HashTable :
                 ptr->write(wb);
     }
 
-    void writeText(DB::WriteBuffer & wb) const
```

Comment:
I am confused a bit why this code is removed.
Probably it's not used, but may be still helpful for debugging.
I would prefer to do it in a separate pr if possible.

Reply:
yeah, let's left it untouched